legal/terms/terms_of_sale.rst

```diff
@@ -0,0 +1,65 @@
+.. _terms_of_sale:
+
+=====================
+General Terms of Sale
+=====================
+
+.. note:: Last revision: May 3, 2017.
+
+These General Terms of Sale govern the sale of products and services by
+Odoo S.A. to you. Additional terms may apply for services provided by Odoo S.A.
+(for example, the Enterprise  Subscription Agreements). If these additional
+terms are inconsistent with the General Terms of Sale, the additional terms will
+supersede these terms.
```

Comment:
I would opt for "the additional terms will prevail over these General Terms of Sale."
Subtle distinction, I know ;-)
Interestingly, [this author](http://www.adamsdrafting.com/control-prevail-take-precedence-or-govern/) also picked "prevail" after comparing some Amazon and Google TOS  ;-)